cmd/internal.go

```diff
@@ -113,9 +122,15 @@ func (r *ParseCommand) ParsePipeline(assetPath string) error {
 		return cli.Exit("", 1)
 	}
 
-	foundPipeline.WipeContentOfAssets()
```

Comment:
@albertobruin Do you know why we need `WipeContentOfAssets` ?

Reply:
maybe because if we don't the size of the json would be too big?